### ISSUE TITLE
exportForHLS not support for mp3

### ISSUE THREAD

**Issue by phuongphally** [protonemedia/laravel-ffmpeg#319]
Hello sir. This library work great for export video to exportForHLS , but not mp3

I wrote 

```php 

 $lowBitrate = (new \FFMpeg\Format\Audio\Aac)->setAudioKiloBitrate(64);
            $midBitrate = (new \FFMpeg\Format\Audio\Aac)->setAudioKiloBitrate(128);
            $highBitrate = (new \FFMpeg\Format\Audio\Aac)->setAudioKiloBitrate(265);

            FFMpeg::fromDisk(config('app.upload_storage'))
                ->open('idol.mp3')
                ->exportForHLS()
                ->setSegmentLength(10) // optional
                ->addFormat($lowBitrate)
                ->addFormat($midBitrate)
                ->addFormat($highBitrate)
                ->toDisk(config('app.stream_storage'))
                ->save('idol.m3u8');

```

Get error 

```json
//Argument 2 passed to ProtoneMedia\\LaravelFFMpeg\\Exporters\\HLSExporter::getSegmentPatternAndFormatPlaylistPath() must be an instance of FFMpeg\\Format\\VideoInterface, instance of FFMpeg\\Format\\Audio\\Mp3 given, called in ..../vendor/pbmedia/laravel-ffmpeg/src/Exporters/HLSExporter.php on line 261


```

**Comment by phuongphally** [comment]
Similar  https://github.com/protonemedia/laravel-ffmpeg/issues/115

**Comment by phuongphally** [comment]
@marbocub can you help me please?

**Comment by phuongphally** [comment]
@pascalbaljet  Please help me.

**Comment by theMSY** [comment]
addFormat is dedicated for video formats!
if you want to modify the audio codec and/or bitrate, use this as an example:
`$highBitrate = (new X264)->setAudioKiloBitrate(320)->setKiloBitrate(1000)->setAudioCodec('aac');`

**Comment by nemesisKO** [comment]
> ```
> ->setSegmentLength(10)
> ```

find a solution?

**Comment by Senya01** [comment]
Any updates?

**Comment by pascalbaljet** [comment]
Fixed in v8.2.1
